### ISSUE TITLE
What subsets are used for training and which subsets are used for testing? ? ?

### ISSUE THREAD

**Issue by limengfei3675** [signatrix/efficientdet#16]
Hello, we tested the pre-trained model you provided and found that we obtained a map of 0.33 on the validation set, and only a map of 0.32 on the training set. Generally, there will be a higher map on the training set. What subsets are used for training and which subsets are used for testing? ? ?

**Comment by wanggrun** [comment]
I am interested in this problem.

**Comment by uvipen** [comment]
We for sure used train2017 for training and val2017 for testing. I agree with you that generally, mAP for training phase should be better than for validation/testing. We are training D1, and will check if we see the same thing happen. Thank you for your report

**Comment by limengfei3675** [comment]
Ok,thanks.
